DesignOps/developmnet-life-cyle.md

```diff
@@ -0,0 +1,328 @@
+
+## This is not a Static Development Journey
+
+Think about it as a Product. We release an MVP and then we iterate to improve it.
+
+
+- Q1. Work in the implementation (Document, templates, productboard changes,communicate to internal stakeholders, etc.)
+- Q2. MVP Run (two sprints)
+  - Expect mistakes, problems, missing thing... learning.
+- Q3. Proper First Run
+  - Run a retrospective at the end of quarter.
+- Q4. Implemente feedback from retrospective.
+
+
+
+## Objectives
+
+#### From HWMs
+
+- HWM have complete and timely product specifications
+- HMW improve suggestion changes coordination with the metadata WG
+- HMW has a seamless handover with precise product specification
+- HMW better incorporate member input?
+- HMW deals with resourcing for the activities in this part of the cycle.
+- HMW successfully coordinate between teams
+- HMW come to a collective agreement on our priorities
+
+
+#### From Affinity Mapping
+
+- Process all insights in ProductBoard
+- Collect Solution Validation Data
+- Prioritisation in the RoadMap meeting
+- Plan broader Communication
+- Collect further input from the community/stakeholders
+- Reviewed feature Technical feasibility
+- Describe what will not be done (or later) and why
+
+## HWMs
+
+
+### HWM have complete and timely product specifications
+
+- Having a checklist of items to include in the spec.
+- Working in parallel tracks with the development team.
+
+### HMW improve suggestion changes coordination with the metadata WG
+
+- Coordinate a campaign of validation for a suggestion that reaches datacite first.
+- Hand over the suggestion and validation results to Metadata WG via the appropriate channels.
+- Publish the metadata WG decision Process in the support website.
+
+### HMW has a seamless handover with clear product specification
+
+- Use a Build/Delivery Backlog meeting every half a sprint to groom use stories in the specification.
+- Use a Sprint Planning meeting before every Sprint to schedule which user stories from the product specification will be built in the Sprint.
+- Asynchronously openly review product specifications and its user stories during the Sprint.
```

Comment:
I'm not sure whether development team is following sprints anymore.

Reply:
ok, yeah I mean any iteration cycle they are following we can probably accomodate. But I guess there would have to be some sort of iterative approach otherwise how would we reach the vision.

Reply:
The Dev Team is using Kanban for work planning.